src/main/java/org/example/managers/TicketManager.java

```diff
@@ -0,0 +1,60 @@
+package org.example.managers;
+
+import org.example.entities.Ticket;
+import org.example.services.TicketService;
+
+import java.util.List;
+import java.util.Optional;
+
+public class TicketManager {
```

Comment:
what is the difference between manager and service classes?

Reply:
service encapsulates the business logic required for the app to perform tasks.  Meanwhile manager class is responsible for coordinating multiple services or operations to handle complex business processes. So in my case it's not right to use manager as additional layer above the service class